sources/packages/backend/package-lock.json

```diff
@@ -83,25 +83,25 @@
         "@types/pg": "^8.11.13",
         "@types/supertest": "^6.0.2",
         "@types/uuid": "^10.0.0",
-        "@typescript-eslint/eslint-plugin": "^5.39.0",
-        "@typescript-eslint/parser": "^5.39.0",
-        "copy-webpack-plugin": "^11.0.0",
+        "copy-webpack-plugin": "^13.0.1",
         "cross-env": "^7.0.3",
-        "eslint": "^8.24.0",
-        "eslint-config-prettier": "7.0.0",
+        "eslint": "^9.36.0",
+        "eslint-config-prettier": "^10.1.8",
         "eslint-plugin-prettier": "^3.1.4",
         "faker": "^5.2.0",
         "find-config": "^1.0.0",
         "jest": "^29.6.4",
+        "jiti": "^2.6.0",
```

Comment:
Why was this package added?

Reply:
To allow the use of a ts file without using the experimental node options for now.
Once the config is changed to ts, there is a warning asking to have this dev-only package added.
https://eslint.org/docs/latest/use/configure/configuration-files#typescript-configuration-files